package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "david_runger",
   "private": true,
+  "type": "module",
```

Comment:
Fixes this warning:

```
❯ CI=true NODE_ENV=production VITE_RUBY_ENTRYPOINTS_DIR=admin_entrypoints VITE_RUBY_PUBLIC_OUTPUT_DIR=vite-admin ./node_modules/.bin/vite build
(node:135261) [MODULE_TYPELESS_PACKAGE_JSON] Warning: Module type of file:///home/david/code/david_runger/tailwind-active_admin.config.js?id=1743206456474 is not specified and it doesn't parse as CommonJS.
Reparsing as ES module because module syntax was detected. This incurs a performance overhead.
To eliminate this warning, add "type": "module" to /home/david/code/david_runger/package.json.
(Use `node --trace-warnings ...` to show where the warning was created)
```

I'm a tiny bit surprised that I never had occasion to add this before?